test_SSL_get_privatekey.c

```diff
@@ -0,0 +1,29 @@
+#include <ssl.h>
```

Comment:
This is my simple mistake. This test code for my local test. Test items are already included into api.c. I'll remove this file from pull request.